main.go

```diff
@@ -220,6 +224,29 @@ func setupReconcilers(mgr ctrl.Manager) {
 	}
 }
 
+func setupWebhooks(mgr ctrl.Manager) {
+	coreStrapProvider := new(webhook.CoreProviderWebhook)
+	if err := coreStrapProvider.SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "CoreProvider")
+		os.Exit(1)
+	}
+	bootStrapProvider := new(webhook.BootstrapProviderWebhook)
+	if err := bootStrapProvider.SetupWebhookWithManager(mgr); err != nil {
```

Comment:
Don't think you need to create the variable like this
```suggestion
	if err := webhook.BootstrapProviderWebhook{}.SetupWebhookWithManager(mgr); err != nil {
```

Reply:
Updated as per the suggestion